### ISSUE TITLE
System Configuration - Email Notifications

### ISSUE THREAD

**Issue by crashff00** [ignacionelson/ProjectSend#148]
There really needs to be a test function for the email notifications setup.

I have the Phpmailer configured currently, and I'm getting no emails at all, even though it says it's working. 

If I were to pick SMTP, or any other type of email setup, there's no way for me to test it without attempting to create a user, and then deleting and recreating every time I need to test a change made.

**Comment by lenamtl** [comment]
Yes why not.
Are you testing locally or live?

**Comment by crashff00** [comment]
I'm testing locally currently.  When I test the smtp relay on my webserver, I get a test notification with no errors, but when I set up SMTP on ProjectSend, it tells me it cant send anything. I don't know what the correct configuration is because of a lack of documentation.

**Comment by lenamtl** [comment]
You won't be able to send email using SMTP without installing something like Stunnel
https://www.stunnel.org/index.html

Here the way I'm setting it up on Windows & Wampserver
http://yogeshchaugule.com/blog/2013/configure-sendmail-wamp

**Comment by crashff00** [comment]
Sendmail is already setup and working outside of ProjectSend. it is NOT getting any emails from ProjectSend.

ProjectSend set as Phpmailer, it says is all good, but no email sent, or even logged.
ProjectSend set as smtp mail it says cant send anything because of errors.

**Comment by crashff00** [comment]
I reset the configuration on everything.  I have set my email notices configuration for the following:

From: admin email
Send copies: When a system user uploads files,  When a client uploads files
Additional copies received by: Address supplied above (on "From"), Also to this addresses: 2nd admin email
Expiration: 2 attempts
Days to expire: 15
Sending Option: PHPMail (Basic)
SMTP/Gmail shared options - blank
SMTP options - blank

When I created a test user, the following happened -   Test user account got their registration email.   The admin email and the 2nd admin email did not receive anything.

**Comment by bobbydback** [comment]
Did you ever find a solution to the issue of not receiving e-mail notifications when files are uploaded by users or client? I'm having the same issue.

**Comment by lenamtl** [comment]
If you have made all changes to allow your system to send mail from local and you are sure it is working.
Then try to change Expiration: 2 attempts to higher number.

But like I said you need to make sure your local system is configure correctly to send email.

**Comment by crashff00** [comment]
I looked at the outgoing mail logs, and the admin email is not even listed as a destination address on the email.  With r583, all the bcc addresses were listed.

The only time I get a copy of an email with 611 is when a new file assignment is made.

**Comment by JeroTwi** [comment]
I use Windows with WAMPserver to host Projectsend. I had an internal SMTP relay problem which I solved by editing line 402 in the send-email.php file (includes/classes directory).

I changed the line:
$this->send_mail->SMTPAuth = true;
To:
$this->send_mail->SMTPAuth = false;

My mailserver had a problem with the constant authentication. By setting this to false it solved the problem for me. I thought I'd share it, maybe this can be helpfull to anyone.
